File: LaGuardia Community College/MAC 125/Class 2/1.cpp
#include <iostream>

int main() {
	int N;
	std::cout << "Enter how many integers you want to input: ";
	std::cin >> N;

	int array1[N];
	int sum = 0;
	for (int x = 0; x < N; x++) {
		std::cout << "Enter number " << x + 1 << "\n";
		std::cin >> array1[x];
		sum = sum + array1[x];
	}

	std::cout << "Sum of numbers: " << sum << "\n"
		<< "Average of numbers: " << sum / N;

	return 0;
}

File: LaGuardia Community College/MAC 283/Assignments/Coding Project/1.py
num1 = input("Enter the number: ")

length = len(num1)
num1 = int(num1)

for i in range(length, 0, -1):
    digit = num1 // (10 ** (i - 1))
    print(digit, end=" ")
    num1 = num1 % (10 ** (i - 1))
